src/main/java/com/stripe/net/RequestOptions.java

```diff
@@ -72,12 +87,21 @@ public boolean equals(Object o) {
     if (clientId != null ? !clientId.equals(that.clientId) : that.clientId != null) {
       return false;
     }
-    if (idempotencyKey != null ? !idempotencyKey.equals(that.idempotencyKey)
-        : that.idempotencyKey != null) {
+    if (idempotencyKey != null ? !idempotencyKey.equals(that.idempotencyKey) :
+        that.idempotencyKey != null) {
+      return false;
+    }
+    if (stripeAccount != null ? !stripeAccount.equals(that.stripeAccount) :
+        that.stripeAccount != null) {
       return false;
     }
-    if (stripeVersion != null ? !stripeVersion.equals(that.stripeVersion)
-        : that.stripeVersion != null) {
```

Comment:
Reordering to match the alphabetically sorted instance variables, plus adding `stripeAccount`